Patches/RH2 Faction - The Rangers/Rangers_CE_Patch_Apparel.xml

```diff
@@ -84,7 +82,7 @@
             <equippedStatOffsets>
                 <PsychicSensitivity>-0.2</PsychicSensitivity>
                 <AimingAccuracy>0.15</AimingAccuracy>
-                <ToxicEnvironmentResistance>0.50</ToxicEnvironmentResistance>
```

Comment:
These should stay as `ToxicEnvironmentResistance`, `ToxicResistance` is used elsewhere and protects against venom bites and other similar damage sources.

Reply:
Changed all to Toxic EnvironmentResistance